### ISSUE TITLE
Wrong starting day of week in calendar while setting an ending date with poll

### ISSUE THREAD

**Issue by JimmyKater** [nextcloud/polls#466]
### What is going wrong?
on my nc14.0.4 I have setup regionals for "germany"... and so my server tells me (and my users) on the setttings page that weeks would begin on mondays, which is correct. 
but when I setup a new poll (in polls 0.9.5) and want to have an ending date on it - the small calendar popping up starts weeks on sundays (which would be correct for like US-EN) - definitly not with monday.

**Expected behavior**
calendar settings in polls should be same as on server.

### Information about your polls installation
**Polls version?** 
0.9.5

**Fresh installation or update from a prior version (from which one)?**
updated from the version before (but never tested any poll)

**How did you install this version?(Appstore or describe installation)**
appstore

### Information about your Instance of Nextcloud/ownCloud
**Nextcloud or ownCloud?**
NC

**Which Version?**
14.0.4.

**Nextcloud/ownCloud version:** (see Nextcloud admin page)
see above

**List of activated apps:**
Enabled:
  - accessibility: 1.0.1
  - activity: 2.7.0
  - admin_audit: 1.4.0
  - admin_notifications: 1.0.2
  - announcementcenter: 3.3.1
  - apporder: 0.6.0
  - audioplayer: 2.5.1
  - bookmarks: 0.15.1
  - bruteforcesettings: 1.3.0
  - calendar: 1.6.4
  - circles: 0.15.2
  - cloud_federation_api: 0.0.1
  - comments: 1.4.0
  - contacts: 2.1.8
  - data_request: 1.1.0
  - dav: 1.6.0
  - dropit: 0.1.3
  - external: 3.1.0
  - federatedfilesharing: 1.4.0
  - federation: 1.4.0
  - files: 1.9.0
  - files_accesscontrol: 1.4.0
  - files_downloadactivity: 1.3.0
  - files_external: 1.5.0
  - files_pdfviewer: 1.3.2
  - files_rightclick: 0.8.4
  - files_sharing: 1.6.2
  - files_texteditor: 2.6.0
  - files_trackdownloads: 1.3.1
  - files_trashbin: 1.4.1
  - files_videoplayer: 1.3.0
  - firstrunwizard: 2.3.0
  - flowupload: 0.0.9
  - gallery: 18.1.0
  - impersonate: 1.1.0
  - issuetemplate: 0.4.0
  - logreader: 2.0.0
  - lookup_server_connector: 1.2.0
  - mail: 0.11.0
  - metadata: 0.8.0
  - nextcloud_announcements: 1.3.0
  - notifications: 2.2.1
  - oauth2: 1.2.1
  - password_policy: 1.4.0
  - passwords: 2018.12.0
  - phonetrack: 0.4.0
  - polls: 0.9.5
  - previewgenerator: 2.0.0
  - provisioning_api: 1.4.0
  - quota_warning: 1.3.0
  - rainloop: 6.0.2
  - ransomware_protection: 1.2.0
  - registration: 0.4.5
  - serverinfo: 1.4.0
  - sharebymail: 1.4.0
  - sharerenamer: 2.3.0
  - socialsharing_email: 1.0.4
  - socialsharing_facebook: 1.0.3
  - socialsharing_twitter: 1.0.3
  - spreed: 4.0.2
  - support: 1.0.0
  - tasks: 0.9.8
  - theming: 1.5.0
  - twofactor_backupcodes: 1.3.1
  - updatenotification: 1.4.1
  - workflowengine: 1.4.0
Disabled:
  - auto_mail_accounts
  - cms_pico
  - dashboard
  - drop_account
  - encryption
  - files_external_gdrive
  - files_snapshots
  - files_versions
  - ownbackup
  - survey_client
  - systemtags
  - user_external
  - user_ldap

```
**Nextcloud configuration:**
root@nextcloud:/var/www/nextcloud# sudo -u www-data php occ config:list system
{
    "system": {
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": {
            "0": "localhost",
            "1": "***REMOVED SENSITIVE VALUE***",
            "2": "nextcloud.local",
            "4": "***REMOVED SENSITIVE VALUE***",
            "5": "***REMOVED SENSITIVE VALUE***",
            "7": "***REMOVED SENSITIVE VALUE***",
            "8": "***REMOVED SENSITIVE VALUE***"
        },
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "overwrite.cli.url": "https:\/\/***REMOVED SENSITIVE VALUE***\/",
        "dbtype": "mysql",
        "version": "14.0.4.2",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "mysql.utf8mb4": true,
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpmode": "smtp",
        "mail_smtpauthtype": "LOGIN",
        "mail_smtpsecure": "ssl",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpauth": 1,
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpport": "465",
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
        "loglevel": 0,
        "maintenance": false,
        "theme": "",
        "updater.release.channel": "production",
        "memcache.local": "\\OC\\Memcache\\Redis",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": 0,
            "timeout": 0,
            "password": "***REMOVED SENSITIVE VALUE***"
        },
        "log_type": "file",
        "defaultapp": "apporder"
    }
}

```
### Server configuration
Operating system: Linux 4.9.35-v7+ #1014 SMP Fri Jun 30 14:47:43 BST 2017 armv7l

Webserver: Apache/2.4.25 (Raspbian) (fpm-fcgi)

Database: mysql 10.1.37

PHP version:

7.2.13-1+0~20181207100540.13+stretch~1.gbpf57305 Modules loaded: Core, date, libxml, openssl, pcre, zlib, filter, hash, Reflection, SPL, sodium, session, standard, cgi-fcgi, mysqlnd, PDO, xml, bz2, calendar, ctype, curl, dom, mbstring, fileinfo, ftp, gd, gettext, iconv, igbinary, imagick, intl, json, ldap, exif, mysqli, pdo_mysql, Phar, posix, readline, redis, shmop, SimpleXML, smbclient, sockets, sysvmsg, sysvsem, sysvshm, tokenizer, wddx, xmlreader, xmlwriter, xsl, zip, libsmbclient, Zend OPcache

Nextcloud version: 14.0.4 - 14.0.4.2

Signing status
Array ( )

Client configuration

Browser: Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:64.0) Gecko/20100101 Firefox/64.0

Operating system: Win 7 pro (64)
**Operating system**:

```
**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
no

### Client configuration
**Device:**
Desktop

**Browser:**
ff 64 (64)
**Operating system:**
Win 7 (64) pro

**Comment by dartcafe** [comment]
Since we use https://github.com/nextcloud/nextcloud-vue this should be done with this PR (https://github.com/nextcloud/nextcloud-vue/pull/146) and a new version of polls.

**Comment by dartcafe** [comment]
Fixed in 1.0 with a slight issue which craates a console log, when first day of week is sunday. Should be solved, when new version of the date picker (nextcloud/nextcloud-vue/pull/783) is implemented.
